.github/actions/run-neoeden-test/actions.yml

```
@@ -0,0 +1,86 @@
+name: 'Run Neo-Eden test workflow'
+description: 'Setup Eden, run go test and publish logs'
+
+inputs:
+  file_system:
+    required: true
+    type: string
+  tpm_enabled:
+    required: true
+    type: bool
+  eve_image:
+    type: string
+  eve_log_level:
+    type: string
+    required: false
+    default: 'info'
+  eve_artifact_name:
+    type: string
+  artifact_run_id:
+    type: string
+  require_virtualization:
+    type: bool
+  docker_account:  # if not provided: use anonymous docker user
+    type: string
+    required: false
+    default: ''
+  docker_token:
+    type: string
+    required: false
+    default: ''
+  aziot_id_scope:
+    description: 'Azure IoT ID scope'
+    required: false
+  aziot_connection_string:
+    description: 'Azure IoT connection string'
+    required: false
+
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Collect Workflow Telemetry
+      uses: catchpoint/workflow-telemetry-action@v2
+      with:
+        proc_trace_sys_enable: true
+        comment_on_pr: false
+    - name: Login to Docker Hub
+      if: inputs.docker_account != ''
+      uses: docker/login-action@v3
+      with:
+        username: ${{ inputs.docker_account }}
+        password: ${{ inputs.docker_token }}
+    - name: Setup Environment
+      uses: ./eden/.github/actions/setup-environment
+      with:
+        file_system: ${{ inputs.file_system }}
+        tpm_enabled: ${{ inputs.tpm_enabled }}
+        eve_image: ${{ inputs.eve_image }}
+        eve_log_level: ${{ inputs.eve_log_level }}
+        eve_artifact_name: ${{ inputs.eve_artifact_name }}
+        artifact_run_id: ${{ inputs.artifact_run_id }}
+        require_virtualization: ${{ inputs.require_virtualization }}
+    - name: Run security tests
+      run: cd tests/sec && go test
+      shell: bash
+      working-directory: "./eden"
```

```suggestion
      run: go test
      shell: bash
      working-directory: "./eden/tests/sec"
```
will it work like this? :thinking: then you don't need to change the directory back

---

Updated it :)